lab_1_classify_by_unigrams/main.py

```diff
@@ -28,6 +43,18 @@ def create_language_profile(language: str, text: str) -> dict[str, str | dict[st
     :param text: a text
     :return: a dictionary with two keys – name, freq
     """
+    if not (
+            isinstance(language, str) and isinstance(text, str)
+    ):
+        return None
+
+    tokens = tokenize(text)
+    frequency_dict = calculate_frequencies(tokens)
+
+    if not isinstance(frequency_dict, dict):
```

Comment:
Given the typing annotation of the `calculate_frequencies`, what could it return if not a dict? Make the check more specific.

Reply:
it returns `None` or a dict. I don't understand what i should change. I did this because i had problems in mypy cheks:(
![image](https://github.com/fipl-hse/2023-2-level-labs/assets/144421718/868e9426-336b-40c9-a127-a07e7052144e)

Reply:
i fixed it!!!